docs/section11/Rule11-15.md

```diff
@@ -0,0 +1,68 @@
+**Function Call:**
+- **get_SWH_uses_associated_with_each_building_segment**  
+- **get_component_by_id**  
+
+**Applicability Checks:**
+- create a list of swh use ids that need to be checked - we'll create the list from both the proposed and baseline models: `swh_use_ids = []`
+- look at each building in the B_RMD: `for building in B_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+    - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - append the swh_use_id to the list of swh use ids: `swh_use_ids.append(swh_use_id)`
+- look at each building in the P_RMD: `for building in P_RMD:`
+    - create a boolean is_applicable and set it to false: `is_applicable = false`
+    - look at each building segment in the building: `for building_segment in building.building_segments:`
+    - get the service water heating uses in the building segment `service_water_heating_use_ids = get_SWH_uses_associated_with_each_building_segment(P_RMD, building_segment.id)`
+    - look at each service water heating use id: `for swh_use_id in service_water_heating_use_ids:`
+        - append the swh_use_id to the list of swh use ids if it's not already in the list: `if !swh_use_id.in sw_use_ids: swh_use_ids.append(swh_use_id)`
+
+- look at each swh_use_id in the swh_use_ids list.  Each of these is applicable (not sure how this works, when an ID is applicable instead of an object): `for swh_use_id in swh_use_ids: CONTINUE TO RULE LOGIC`
+- if there are no swh_use_ids in the list, the rule is not applicable: `if len(swh_use_ids) == 0: NOT_APPLICABLE`
+
+    ## Rule Logic:
+    - get the proposed swh use: `swh_use_p = get_component_by_id(swh_use_id, P_RMD)`
+    - get the proposed swh use: `swh_use_b = get_component_by_id(swh_use_id, B_RMD)`
+    - create a value to track the rule status: `rule_status = "PASS"`
+    - create a variable to track any notes: `rule_note = ""`
+    - if swh_use_b is null: `if swh_use_b == nil:`
+        - set the rule status to FAIL and provide note: `rule_status = "FAIL"; note = swh_use_id + "exists in the Proposed model, but not in the baseline.`
+        - continue to rule assertion: `CONTINUE TO RULE ASSERTION`
+    - if swh_use_P is null: `if swh_use_P == nil:`
+        - set the rule status to FAIL and provide note: `rule_status = "FAIL"; note = swh_use_id + "exists in the Baseline model, but not in the proposed.`
+        - continue to rule assertion: `CONTINUE TO RULE ASSERTION`
+    - check that the use matches by checking that the use, use_units, use_multiplier_schedule, and temperature_at_fixture all match (ARE THERE OTHERS THAT NEED TO MATCH TOO?)
```

Comment:
if we want to check whether they are identical I think we would also need tocheck that the 
 ServiceWaterHeatingDistributionSystem.entering_water_mains_temperature_schedule and the ServiceWaterHeatingDistributionSystem..design_supply_water_temperature, I recall making this comment previously so maybe we discussed it and decide not to?